.github/workflows/ci_build.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           git fetch origin ${{ github.base_ref }} # Fetch target branch to FETCH_HEAD for code style check.
           python3 scripts/build.py --skip-tests --config ${{ matrix.config.type }} --check-code-style-base FETCH_HEAD
```

Comment:
Why do we skip on release? They should run even faster and also flush out mistakes like putting code with side effects inside debug assertions.